shared/chat/conversation/attachment-fullscreen/page.tsx

```diff
@@ -1,16 +1,15 @@
 import * as React from 'react'
 import * as C from '@/constants'
 
-const Full = React.lazy(async () => import('.'))
-type OwnProps = C.ChatProviderProps<C.ViewPropsToPageProps<typeof Full>>
-
-const getOptions = () => ({
```

Comment:
the function form of this causes these objects to be new and cause a render thrash, we already support it being an object and we can also use `useRoute` so i changed most of it thats simple